testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/BulkInsertTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it.http.graphql.graphqlfirst;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.jayway.jsonpath.JsonPath;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(CqlSessionExtension.class)
+public class BulkInsertTest extends GraphqlFirstTestBase {
+
+  private static GraphqlFirstClient CLIENT;
+  private static String KEYSPACE;
+  private static CqlSession SESSION;
+
+  @BeforeAll
+  public static void setup(
+      StargateConnectionInfo cluster, @TestKeyspace CqlIdentifier keyspaceId, CqlSession session) {
+    CLIENT =
+        new GraphqlFirstClient(
+            cluster.seedAddress(), RestUtils.getAuthToken(cluster.seedAddress()));
+    KEYSPACE = keyspaceId.asInternal();
+    SESSION = session;
+    CLIENT.deploySchema(
+        KEYSPACE,
+        "type User @cql_input {\n"
+            + "  id: ID!\n"
+            + "  name: String\n"
+            + "  username: String\n"
+            + "}\n"
+            + "type Query { user(id: ID!): User }\n"
+            + "type InsertUserResponse @cql_payload {\n"
+            + "  applied: Boolean!\n"
+            + "  user: User!\n"
+            + "}\n"
+            + "type Mutation {\n"
+            + "  bulkInsertUsers(users: [UserInput!]): [User]\n"
```

Comment:
Sorry but as I see the examples I'm having second thoughts about response payloads: it looks like `ifNotExists` will work with multiple inputs, so we'd need a way to return (entity + applied) for each input after all.

We should probably allow lists of all the possible single-entity results: list of booleans, list of entities, or list of payloads.

Reply:
done, @olim7t please take a look